osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ClearAlarmRegisterCommandExecutor.java

```diff
@@ -136,4 +166,23 @@ public AccessResultCode executeForAlarmRegister(
       throw new ConnectionException(e);
     }
   }
+
+  public Optional<AttributeAddress> findAttributeAddress(
```

Comment:
Should this function be private?
Or should it be moved to a more general place? Maybe more command executors need a function like this.

Reply:
Is moved to ObjectConfigServiceHelper